src/components/SelectInput/SelectInput.jsx

```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames';
 import Select from 'react-select';
 import FormLabel from '../FormLabel/FormLabel';
 import InputValidationMessage from '../InputValidationMessage/InputValidationMessage';
-import './SelectInput.scss';
+import './ReactSelect.scss';
```

Comment:
Hey @JCarmine the reason I named it ReactSelect is because these styles are applied to the actual ReactSelect component which happens to be housed in our own SelectInput. The ReactSelect also has to be a global to work properly with the prefixing that react-select uses. Perhaps the best way is to move this to the /globals folder, but to be clear I would still like us to have a module for SelectInput so that any additional styling that doesn't belong to react-select can be done there in the future.